File: android/app/src/main/java/com/collectrndemo/modules/collect/field/date/CardExpirationDateModule.java
package com.collectrndemo.modules.collect.field.date;

import androidx.annotation.NonNull;

import com.facebook.react.bridge.ReactApplicationContext;
import com.facebook.react.bridge.ReactContextBaseJavaModule;

public class CardExpirationDateModule extends ReactContextBaseJavaModule {

    public CardExpirationDateModule(ReactApplicationContext reactContext) {
        super(reactContext);
    }

    @NonNull
    @Override
    public String getName() {
        return "Collect " + this.getClass().getSimpleName();
    }
}

File: android/app/src/main/kotlin/com/collectrndemo/simple/modules/collect/field/date/CardExpirationDateModule.kt
package com.collectrndemo.simple.modules.collect.field.date

import com.facebook.react.bridge.ReactApplicationContext
import com.facebook.react.bridge.ReactContextBaseJavaModule

class CardExpirationDateModule(
    reactContext: ReactApplicationContext
) : ReactContextBaseJavaModule(reactContext) {

    override fun getName(): String = "Collect " + this.javaClass.simpleName
}
